WcaOnRails/app/views/persons/_details.html.erb

```diff
@@ -1,6 +1,13 @@
 <div>
   <div class="text-center">
-    <h2><%= @person.name %></h2>
+    <h2>
+      <%= @person.name %>
+      <% if current_user.person == @person %>
+        <%= link_to [:edit, @current_user] do%>
+          <%= content_tag :i, "", :class => "fa fa-edit" %>
+        <% end %>
```

Comment:
This is actually not true in all cases. If an admin accesses this page, we want to link him to this person's profile rather than his own =) Also there is a neat `icon` helper method for FontAwesome icons! I'd suggest doing it like so:
```ruby
<%= link_to icon("edit"), edit_user_path(@person.user) %>
```